src/Sulu/Bundle/AdminBundle/Resources/public/scss/modules/login.scss

```diff
@@ -1,187 +1,270 @@
-        }
 
-        .bottom-container {
             position: absolute;
+            z-index: 10;
             bottom: 0px;
             left: 0px;
-            width: 100%;
-            padding: 0 50px;
-            @include clear-float;
-
-            span {
-                display: block;
-                padding: 10px 0 10px 10px;
-                cursor: pointer;
-                color: $grayLight;
-                float: right;
-                &:hover {
-                  color: $black;
-                }
-            }
-            .website-switch {
-                float: left;
-                padding-left: 0;
-            }
-        }
 
-        .reset-mail {
-            .bottom-container span:not(.website-switch) {
-                color: $black;
-            }
+            text-align: center;
         }
 
-        .reset-mail-message {
-            text-align: center;
-            .message {
-                display: block;
-                color: $grayLight;
-                padding-bottom: 20px;
-            }
-            .btn.action {
-              float: none;
-              margin: 0;
-              padding: 0 15px;
+        .success-overlay {
+            height: 100%;
+            width: 100%;
+            position: relative;
+            z-index: -10;
+
+            background-color: $green;
+
+            // success animation
+            opacity: 0;
+            transition: opacity 300ms;
+
+            .success-icon {
+                @include centerBlock;
+                font-size: 200px;
+                color: $white;
             }
         }
 
-        label {
-            text-align: center;
-            padding-bottom: 5px;
+        .login-loader {
+            display: block;
+            position: absolute;
+            top: 0;
+            right: 0;
+            z-index: -10;
         }
     }
 
-    .background {
-        display: none;
-        position: absolute;
+    &.content-success{
+        .success-overlay {
+            opacity: 1;
+        }
 
-        .image {
-            position: relative;
-            display: block;
-            width: 100%;
-            height: 100%;
-            @include background-size(cover);
-            background-repeat: no-repeat;
-            background-position: center center;
-            @include blur;
-            z-index: 10;
+        .media-container {
+            left: -50%;
         }
 
-        .darkener {
-            position: absolute;
-            top: 0px;
-            left: 0px;
-            display: block;
-            width: 100%;
-            height: 100%;
-            background: #000;
-            opacity: 0.35;
-            transition: opacity 450ms;
-            z-index: 20;
+        .content-container {
+            left: 50%;
         }
+    }
+}
 
-        &.active {
-            .darkener {
-                opacity: 0.15;
-            }
+.login-label-container {
+    // center labels in content-container
+    left: 75% !important;
```

Comment:
@nnatter isn't there a better way than using !important. Try to use a stronger selector